src/test/java/org/gridsuite/shortcircuit/server/ShortCircuitAnalysisControllerTest.java

```diff
@@ -218,7 +218,7 @@ private static void assertFaultResultsEquals(List<FaultResult> faultResults, Lis
             assertEquals(orderedFeederResultsDto.size(), orderedFeederResults.size());
             for (int j = 0; j < orderedFeederResultsDto.size(); j++) {
                 assertEquals(orderedFeederResultsDto.get(j).getConnectableId(), orderedFeederResults.get(j).getConnectableId());
-                if (orderedFeederResults.get(i) instanceof MagnitudeFaultResult) {
+                if (faultResults.get(i) instanceof MagnitudeFaultResult) {
                     assertEquals(orderedFeederResultsDto.get(j).getCurrent(), ((MagnitudeFeederResult) orderedFeederResults.get(j)).getCurrent(), 0.1);
                 } else if (faultResults.get(i) instanceof FortescueFaultResult) {
                     assertEquals(faultResultsDto.get(i).getPositiveMagnitude(), ((FortescueFaultResult) faultResults.get(i)).getCurrent().getPositiveMagnitude(), 0.1);
```

Comment:
I reckon we ought to also fix this bit  `faultResultsDto.get(i)` should be `orderedFeederResultsDto.get(j)` and  `faultResults.get(i)` should be `orderedFeederResults.get(j)` otherwise the test loop on the same data

Reply:
and actually this test is already done line 203 isn't it ?

Reply:
Yes the oversight must have occurred during a bad case of copy pasta

Reply:
```suggestion
                    assertEquals(orderedFeederResultsDto.get(j).getPositiveMagnitude(), ((FortescueFeederResult) orderedFeederResults.get(j)).getCurrent().getPositiveMagnitude(), 0.1);

```